Makefile

```diff
@@ -0,0 +1,37 @@
+include pandoc-man.mk
+
+ifeq ($(PREFIX),)
+    PREFIX := /usr/local
+endif
+
+datarootdir := $(PREFIX)/share
+datadir := $(datarootdir)
+mandir := $(datarootdir)/man
+bindir := $(datarootdir)/bin
+
+
+all: quickget.1 quickemu.1 quickemu_conf.1
+
+clean:
+	rm *.1
+
+install: all
+	install -d $(DESTDIR)$(mandir)/man1
+	install -d $(DESTDIR)$(bindir)
+	#install -m 644 quickget.1 quickemu.1 quickemu_conf.1 $(DESTDIR)$(mandir)/man1
+	install -m 644 quickget.1 $(DESTDIR)$(mandir)/man1
+	install -m 644 quickemu.1 $(DESTDIR)$(mandir)/man1
+	install -m 644 quickemu_conf.1 $(DESTDIR)$(mandir)/man1
+	#install -m 644 quickgui.1 $(DESTDIR)$(mandir)/man1
+
+
+uninstall::
```

Comment:
Why are there two `:`? Is this some Makefile magic I am too noob to understand, or a simple typo? XD
(There are two `:` in the commented out `clean` as well, in case this was an error)

Reply:
They are different. see [here](https://www.gnu.org/software/make/manual/html_node/Double_002dColon.html). As I am a noob who vaguely understands makefiles and spends a lot of time borrowing examples and then testing them until my improvements stop spraying files in all the wrong places I probably need to read the manual a few more times before I get the differences. So not a typo but possibly a different source of inspiration rather than actual intent.